### PR TITLE
Manipulate Requestability

### PR DESCRIPTION
**Pull request by nodanaonlyzuul**

### CODE REVIEW

File: test/requestability_determination.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+var requestableBasedOnStatusAndHoldingLocation = require('../lib/requestability_determination').requestableBasedOnStatusAndHoldingLocation
+
```
Comment (nodanaonlyzuul) [inline test/requestability_determination.test.js]: Test the corner case of SCSB saying something is not available but our serialized status saying it is.

File: lib/requestability_determination.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const sierraLocations = require('@nypl/nypl-core-objects')('by-sierra-location')
+const statusMapping = require('@nypl/nypl-core-objects')('by-statuses')
+
+// This method is not meant to be used in isolation.
+// The logic of what makes something requestable is pretty complex.
+// As of writing - this method is only meant to be called on recap items owned by NYPL
+let requestableBasedOnStatusAndHoldingLocation = (item) => {
+  // only consider flipping something if our index says it's requestable.
+  // Never turn an unrequestable item to requestable = [true]
```
Comment (nodanaonlyzuul) [inline lib/requestability_determination.js]: This needs to change to ALWAYS manipulate requestable, but also, consider the SCSB response's about availability.
